src/modules/sensors/sensor_params.c

```diff
@@ -3258,3 +3305,108 @@ PARAM_DEFINE_INT32(PWM_AUX_DISARMED, 1000);
  * @group PWM Outputs
  */
 PARAM_DEFINE_FLOAT(MOT_SLEW_MAX, 0.0f);
+
+/**
+ * Define the sysID manoeuvre
+ *
+ * @min 0
+ * @max 7
+ * @value 0 Disabled
+ * @value 1 Step/Ramp in roll
+ * @value 2 Step/Ramp in pitch
+ * @value 3 Step/Ramp in yaw
+ * @value 4 Step in throttle
+ * @value 5 Chirp in roll
+ * @value 6 Chirp in pitch
+ * @value 7 Chirp in yaw
+ * @value 8 2-1-1 in roll
+ * @value 9 2-1-1 in pitch
+ * @value 10 2-1-1 in yaw
+ * @group SysID
+ */
+PARAM_DEFINE_INT32(SID_MANOEUVRE, 0);
```

Comment:
There is a max of 7 and a min of 0.  What values should be used for the 2-1-1 maneuvers?